internal/storage/storageutil/object_util.go

```diff
@@ -0,0 +1,58 @@
+package storageutil
+
+import (
+	"crypto/md5"
+	"time"
+
+	"cloud.google.com/go/storage"
+	"github.com/jacobsa/gcloud/gcs"
+	storagev1 "google.golang.org/api/storage/v1"
+)
+
+func ObjectAttrsToBucketObject(attrs *storage.ObjectAttrs) *gcs.Object {
+	// Converting []ACLRule returned by the Go Client into []*storagev1.ObjectAccessControl which complies with GCSFuse type.
+	var Acl []*storagev1.ObjectAccessControl
```

Comment:
variable names should start with lowercase letter and follow camelcase

Reply:
fixed